staging/provisioning/windows/kubeletstart.ps1

```diff
@@ -227,7 +227,7 @@ if ($global:NetworkPlugin -eq "kubenet") {
             $process | Stop-Process | Out-Null
         }
 
-        ./cleanupnetwork.ps1 
+        & "c:\k\cleanupnetwork.ps1"
```

Comment:
should we also update the other places this is called?

https://github.com/Azure/aks-engine/blob/1d2b73ca07624d0e5f1ce4f16a604c0de2f52091/staging/provisioning/windows/windowsnodereset.ps1#L52

https://github.com/Azure/aks-engine/blob/d8807cb67c8b157399b43826c89041a615f877b8/staging/provisioning/windows/kubeletstart.ps1#L194